docker/dev/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6-alpine
+
+RUN apk update && apk add python3-dev build-base git
+
+WORKDIR /app
+
+# Cache the dependencies
+COPY setup.py .
+RUN mkdir xain_fl && \
+    printf '__version__ = "0.2.0"\n__short_version__ = "0.2"' > xain_fl/__version__.py && \
```

Comment:
Do we need to update the version values when we release a new package or are they just dummy values for the `setup.py`?

Reply:
They're just dummy values for the `setup.py` indeed.